### DISCUSSION TITLE
V 10533, where's option to patch any other boot img?

### DISCUSSION THREAD

**Discussion by invisible789** [bmax121/APatch#282] [Q&A] [answered]
10533 is working great, i want to patch another boot img and not finding option to do so, i may be lazy enough to locate option, 😕
![Screenshot_20240311-044026_crDroid Home](https://github.com/bmax121/APatch/assets/72073975/91523985-f1d4-4849-9767-b8f2d0075647)

**Comment by pomelohan** [comment]
Wait for new release, we had refactored the installation.

**Comment by pomelohan** [comment, marked as answer]
The new version has been released.

**Comment by invisible789** [comment]
Resolved with new version
